README.md

```diff
@@ -128,7 +128,7 @@ The same as the Guild Highlighter, except for members of the Card Gardening
 Society.
 
 #### [Main Auction Displayer](userscripts/auction/Main%20Auction%20Displayer.user.js)
-Contacts a secret Google Spreadsheet with known puppets and puppetmasters. Adds
```

Comment:
It is important to stress to users that the so-called secret spreadsheet is not secret at all. I suggest stressing that instead of just removing the word "secret", ex. change it to "not-secret" or "totally not secret" or "_[…]_ that was never ever secret".